### PR TITLE
chore: migration to create auth_tokens table

### PR DESCRIPTION
**Pull request by lemald**
#### Summary of changes
**Asana Ticket:** [🏹 figure out authentication for existing arrow API](https://app.asana.com/0/584764604969369/1173297500981963/f)

Putting the migration in a separate PR so we can deploy it first, _then_ the code that actually uses the table.

#### Reviewer Checklist
- [ ] Meets ticket's acceptance criteria
- [ ] Any new or changed functions have typespecs
- [ ] Tests were added for any new functionality (don't just rely on Codecov)
- [ ] This branch was deployed to the staging environment and is currently running with no unexpected increase in warnings, and no errors or crashes.

### CODE REVIEW

File: priv/repo/migrations/20200713155611_create_auth_tokens.exs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+defmodule Arrow.Repo.Migrations.CreateAuthTokens do
+  use Ecto.Migration
+
+  def change do
+    create table(:auth_tokens) do
+      add :username, :string, null: false
```
Comment (losvedir) [inline priv/repo/migrations/20200713155611_create_auth_tokens.exs:6]: I _think_ `:string` maps to `VARCHAR(255)`, while `:text` is preferred for postgres. Performance wise there's no difference, but `text` is more flexible (and can have a length constraint added or changed later.

Comment (losvedir) [reply priv/repo/migrations/20200713155611_create_auth_tokens.exs:6]: Same for `:token`.

Comment (lemald) [reply priv/repo/migrations/20200713155611_create_auth_tokens.exs:6]: Ah, after fiddling around a bit with `mix ecto.migrate --log-sql` I think you're right.

**Comment by losvedir** [comment]
Sorry, just remembered something and wanted to get that comment in, to prevent the merge. Can you also update the `priv/repo/structure.sql` file, via `mix ecto.dump`?

**Comment by lemald** [comment]
@losvedir Good catch! Just pushed the update.

**Comment by digitalcora** [comment]
> Can you also update the `priv/repo/structure.sql` file, via `mix ecto.dump`?

We can make this happen automatically by creating appropriate aliases for `ecto.migrate` and `ecto.rollback`. [Example of this on alerts_concierge](https://github.com/mbta/alerts_concierge/blob/master/mix.exs#L38).

**Pull request merged**